drivers/storage/portworx/components_test.go

```diff
@@ -12911,6 +12904,26 @@ func TestPodDisruptionBudgetWithMetroDR(t *testing.T) {
 	err = testutil.Get(k8sClient, storagePDB, component.StoragePodDisruptionBudgetName, cluster.Namespace)
 	require.NoError(t, err)
 	require.Equal(t, 2, storagePDB.Spec.MinAvailable.IntValue())
+
+	// When portworx version is lesser than 3.1.1 , cluster domain field will not be available, schedulerNodeName will be used to decide the nodes in current domain
```

Comment:
Let's duplicate this test case and test scenario
- when NodeLabelPortworxVersion has invalid version string and the parsing fails
- when the version is greater than or equal 3.1.0 and less than 3.1.1

Basically we should try to cover all the branches in our code for good UT coverage.